.travis.yml

```diff
@@ -19,24 +19,24 @@ cache:
     - lighthouse-viewer/node_modules
     - /home/travis/.rvm/gems/
 install:
+  - export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
   - yarn
   # travis can't handle the parallel install (without caches)
   - yarn run install-all:task:windows
 before_script:
   - export DISPLAY=:99.0
-  - export CHROME_PATH="$(pwd)/chrome-linux/chrome"
+  - export CHROME_PATH="$(which google-chrome-stable)"
   - sh -e /etc/init.d/xvfb start
   - yarn build-all
 script:
-  - echo $TRAVIS_EVENT_TYPE;
-  - echo $TRAVIS_BRANCH
   - yarn bundlesize
   - yarn lint
   - yarn unit
   - yarn type-check
   - yarn closure
   - yarn smoke
   - yarn smokehouse
+  - node lighthouse-extension/test/extension.js
```

Comment:
let's add an npm scripts entry to the extensions package.json so it's easier to run on demand

    cd lighthouse-extension; yarn test:e2e